openmapflow/scripts/deploy.sh

```diff
@@ -17,11 +17,14 @@ env | grep OPENMAPFLOW
 echo "2/8 Ensuring latest models are available for deployment"
 dvc pull "$OPENMAPFLOW_MODELS_DIR".dvc -f
 
-export OPENMAPFLOW_MODELS=$(
-        python -c \
-        "from openmapflow.config import get_model_names_as_str; \
-        print(get_model_names_as_str())"
-)
+if [[ -z "$OPENMAPFLOW_MODELS" ]]; then
```

Comment:
it would be nice if we could do something like `openmapflow deploy <my_models>` to deploy a specific set of models in the future

Reply:
Ah yes, but you run into bash shell fun stuff: https://unix.stackexchange.com/questions/278544/how-to-pass-array-to-bash-shell-script

So I think this will do for now:
```
export OPENMAPFLOW_MODELS=<your models>
openmapflow deploy
```